README.md

```diff
@@ -89,35 +109,49 @@ import { validate } from 'ember-validity-modifier';
 
 export default class MyForm extends Component {
   @action
-  async validateAndSubmit(event) {
-    let { target: form } = event;
-    event.preventDefault();
-    await validate(...form.elements);
-    if (form.reportValidity()) {
-      console.log('Fake submit action', Object.fromEntries(new FormData(form)));
-    }
+  validateAndSubmit({ target: form }) {
+    console.log('Fake submit action', Object.fromEntries(new FormData(form)));
   }
 }
 ```
 
-#### Example with `validator-update` event
+#### Example with `validateDidInsert` argument
 
-To validate the form state on initial render and any time its dependent arguments change, add the `'validator-update'` event to the list of events passed in via the `on` argument.
+To validate the form state on initial render add `validateDidInsert=true`.
```

Comment:
Is there another name that'd be better suited here?
I think it'd be great if render lifecycle was totally abstracted away from the end user

Reply:
`validateFirstRender`?

Reply:
maybe sub-object?
```
validate=(hash
  eagerly=true
)
```

Reply:
Almost the same as the `on=` parameter.

`validateImmediatly=true`?